models/preles/R/runPRELES.jobsh.R

```diff
@@ -15,21 +15,19 @@
 ##' @param end_date End time of the simulation
 ##' @export 
 ##' @author Tony Gardella, Michael Dietze
-runPRELES.jobsh<- function(met.file,outdir,start.date,end.date){
+runPRELES.jobsh<- function(met.file,outdir,priors,start.date,end.date){
   
-  if(!require("PEcAn.utils")) print("install PEcAn.utils")
-  require("lubridate")
+  require("PEcAn.all")
```

Comment:
This is a bad dependency. First, it will slow down the run a LOT since every single ensemble member will have to load the entirety of PEcAn only to use <0.1% of that code. Second, it will make it very hard to run PRELES remote. We're trying to keep dependencies to a minimum to make it easier to do 'light' installs of just a few packages on cluster nodes. You should trim this down to just the parts you need.